dev/src/sample.test.html

```diff
@@ -1,432 +1,560 @@
 <!DOCTYPE html>
-<html lang="en" dir="ltr">
+<html lang="en" dir="ltr" class="su-smooth-scroll">
```

Comment:
Have you looked at this plugin https://github.com/innocenzi/tailwindcss-scroll-snap

How does this differ?

Reply:
The plugin might be useful but the smooth scroll is different. I've used it for the Giving site which enables that scrolling animation when click on anchor links - seems like that's common now.